dev-tools/omdb/src/bin/omdb/db/db_metadata.rs

```diff
@@ -27,8 +31,37 @@ pub struct DbMetadataArgs {
 
 #[derive(Debug, Subcommand, Clone)]
 pub enum DbMetadataCommands {
+    /// Lists the `db_metadata_nexus` records for all Nexuses.
     #[clap(alias = "ls-nexus")]
     ListNexus,
+
+    /// !!! DANGEROUS !!! Updates a `db_metadata_nexus` record to 'Quiesced'
+    ///
+    /// THIS OPERATION IS DANGEROUS. It is the responsibility of the caller
+    /// to ensure that the specified Nexus zone is not running.
+    ///
+    /// If the Nexus being updated is actually running, this operation
+    /// may cause arbitrary data corruption, as it can allow multiple Nexuses
+    /// at distinct database verions to inadvertently be running concurrently.
+    ///
+    /// This operation is intended to assist in the explicit case where a Nexus
+    /// is unable to finish marking itself quiesced during the handoff process,
+    /// and cannot be expunged.
+    ForceNexusQuiesce(ForceNexusQuiesceArgs),
```

Comment:
Take it or leave it:

```suggestion
    ForceMarkNexusQuiesced(ForceMarkNexusQuiescedArgs),
```

The idea is to convey that you're _marking_ Nexus quiesced, whether it is or not -- and _not_ that you're actually quiescing anything.

Reply:
Updated!